tests/files/rbdigital/response_catalog_all_sample.json

```diff
@@ -18,15 +18,15 @@
     "images": [
       {
         "name": "small",
-        "url": "http://images.oneclickdigital.com/1746552/1746552_image_71x108.jpg"
+        "url": "http://images.rbdigital.com/1746552/1746552_image_71x108.jpg"
```

Comment:
We don't use these URLs for real, but they need to stay the same -- they still work and I don't know what the new ones would be.